cmd/sysbox-fs/main.go

```diff
@@ -310,6 +310,18 @@ func main() {
 
 		logrus.Info("Initiating sysbox-fs ...")
 
+		// Print key configuration knobs settings.
+		if ctx.BoolT("allow-immutable-remounts") {
+			logrus.Info("Initializing with 'allow-immutable-remounts' enabled")
```

Comment:
Suggest to make it shorter: "allow-immutable-remounts = enabled"

Reply:
Hmm, not sure, id 'rather keep it like this.